advanced/Timeseries stitching demo.ipynb

```diff
@@ -0,0 +1,1200 @@
+       "      <td>1558048972229913</td>\n",
+       "      <td>&lt;NA&gt;</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>heatSensor:a513a7b6-a67b-440a-b6b0-0bd1188b175b</td>\n",
+       "      <td>heatSensor</td>\n",
+       "      <td>A heat sensor unit</td>\n",
+       "      <td>{}</td>\n",
+       "      <td>time_stitching_demo</td>\n",
+       "      <td>7026854450382658</td>\n",
+       "      <td>1586264971236</td>\n",
+       "      <td>1586264971236</td>\n",
+       "      <td>1558048972229913</td>\n",
+       "      <td>1558048972229913</td>\n",
+       "      <td>widgetHeatSensor:22b8de78-0a2e-4930-b1a8-f7315...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>heatSensor:985ccacb-1594-4ea0-9bd3-91986115a1c8</td>\n",
+       "      <td>heatSensor</td>\n",
+       "      <td>A heat sensor unit</td>\n",
+       "      <td>{}</td>\n",
+       "      <td>time_stitching_demo</td>\n",
+       "      <td>7049557553599949</td>\n",
+       "      <td>1586264971236</td>\n",
+       "      <td>1586264971236</td>\n",
+       "      <td>1558048972229913</td>\n",
+       "      <td>1558048972229913</td>\n",
+       "      <td>widgetHeatSensor:22b8de78-0a2e-4930-b1a8-f7315...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                          externalId              name  \\\n",
+       "0  widgetHeatSensor:22b8de78-0a2e-4930-b1a8-f7315...  widgetHeatSensor   \n",
+       "1    heatSensor:a513a7b6-a67b-440a-b6b0-0bd1188b175b        heatSensor   \n",
+       "2    heatSensor:985ccacb-1594-4ea0-9bd3-91986115a1c8        heatSensor   \n",
+       "\n",
+       "                  description metadata               source                id  \\\n",
+       "0  A heat sensor for a widget       {}  time_stitching_demo  1558048972229913   \n",
+       "1          A heat sensor unit       {}  time_stitching_demo  7026854450382658   \n",
+       "2          A heat sensor unit       {}  time_stitching_demo  7049557553599949   \n",
+       "\n",
+       "     createdTime  lastUpdatedTime            rootId          parentId  \\\n",
+       "0  1586264971236    1586264971236  1558048972229913              <NA>   \n",
+       "1  1586264971236    1586264971236  1558048972229913  1558048972229913   \n",
+       "2  1586264971236    1586264971236  1558048972229913  1558048972229913   \n",
+       "\n",
+       "                                    parentExternalId  \n",
+       "0                                                NaN  \n",
+       "1  widgetHeatSensor:22b8de78-0a2e-4930-b1a8-f7315...  \n",
+       "2  widgetHeatSensor:22b8de78-0a2e-4930-b1a8-f7315...  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
I was being paranoid about people forgetting to clean up and using source to indicate where it came from; but if you are not that worried about that, then I would agree removing it would simplify things. 👍